README.md

```diff
@@ -1,34 +1,58 @@
-```
-| > open-liberty-tools-vscode
-| v liberty-language-server
-| | > lemminx-liberty
-| | > liberty-ls
-```
+![Screenshot of Liberty feature suggestions in a feature block in a server.xml file](./docs/images/feature-completion.png "Completion suggestions for Liberty configuration in server.xml")
 
-2. In the `open-liberty-tools-vscode` directory, run `npm run build`.
+### Hover on Liberty server configuration files
+* Hover for Liberty properties and variables
 
-3. Open the debug view, select and launch `Run Extension (open-liberty-tools-vscode)`. It will open a new window with the extension running in debug mode.
+![Screenshot of a documentation dialog appearing when hovering over a Liberty property in a bootstrap.properties file](./docs/images/property-hover.png "Hover on Liberty properties in bootstrap.properties")
+![Screenshot of a documentation dialog appearing when hovering over a Liberty variable in a server.env file](./docs/images/variable-hover.png "Hover on Liberty server variables in server.env")
+* Hover for Liberty XML configs
 
-4. In the same debug view, now select and launch one of the following to debug each respective project:
-    * `Debug attach liberty-lemminx`
-    * `Debug attach liberty-ls` (in progress)
\ No newline at end of file
+![Screenshot of feature documentation appearing when hovering over a Liberty feature in a server.xml file](./docs/images/feature-hover.png "Hover on Liberty features in server.xml")
+
+### Diagnostics on Liberty server configuration files
+* Diagnostics on Liberty properties and variables
+
+![Screenshot showing diagnostics marking an invalid value for a Liberty property in a bootstrap.properties file. Hovering over the diagnostic will provide more details.](./docs/images/property-diagnostic.png "Diagnostics on Liberty properties in bootstrap.properties")
+![Screenshot showing diagnostics marking an invalid value for a Liberty variable in a server.env file. Hovering over the diagnostic will provide more details.](./docs/images/variable-diagnostic.png "Diagnostics on Liberty variables in server.env")
+* Diagnostics for Liberty XML configs
+
+![Screenshot showing diagnostics marking an invalid feature defined in a server.xml file. Hovering over the diagnostic will provide more details.](./docs/images/feature-diagnostic.png "Diagnostics on Liberty features in server.xml")
+
+### Liberty dev mode schema generation
+If [Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven) or [Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle) is configured with the Liberty project, Liberty Config Language Server will automatically generate a schema file based on the Liberty runtime and version to provide relevant information about the supported `server.xml` elements and Liberty features.
+
+> ### Note for dev mode for containers (devc)
```

Comment:
You can't refer to it as `devc` as that only applies to LMP. Need to spell out "dev mode for containers".